### PR TITLE
feat: home page above fold + refresh of many elements

### PR DESCRIPTION
**Pull request by marbemac**
- Still have to cleanup the content below the fold on the landing page - will do in separate PR if there is time. 
- Will do enterprise page and cleanup of product/use-case pages in separate PRs.
- I probably didn't update everywhere needed re the netlify admin UI, and not everything that I added is customizable via the netlify admin UI yet. We can tackle this in a separate PR after tomorrow's release.
- Mobile / responsive needs to be addressed in, you guessed it, a separate PR ;). This PR is already big enough.

### CODE REVIEW

File: netlify/landings/lp-design.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ hero:
   short: true
   ctas:
     - href: '#demo'
-      name: Get a free demo
+      title: Get a free demo
```
Comment (ssspear) [inline netlify/landings/lp-design.yaml]: I thought we are using the verbiage `Book a Demo`. Is this verbiage on the `/lp` pages for another reason?

File: src/components/Header/Desktop.tsx [hunk 2]
```diff
@@ -60,51 +76,180 @@ const HeaderButton: React.FunctionComponent<IHeaderItem> = ({ title, href, icon,
...
+};
+
+const ProductLinks = () => {
+  return (
+    <div className="p-6">
+      <div className="uppercase font-medium text-grey-darker ml-2">For Developers & Technical Writers</div>
+
+      <div className="flex mt-3">
+        <ProductLink
+          name="Stoplight Studio"
```
Comment (ssspear) [inline src/components/Header/Desktop.tsx]: Are we no longer using the yaml files for the content?

Comment (marbemac) [reply src/components/Header/Desktop.tsx]: I simply didn't have time to get the whole netlify thing set up - dropping a note in the PR description - we'll probably want to tackle the netlify admin UI in a followup PR after tomorrow.

**Comment by ssspear** [comment]
noticed a couple of other issues while perusing 


**styling on about us page**

Screen Shot 2019-08-19 at 2 43 12 PM

**/mocking isn't loading**

Screen Shot 2019-08-19 at 2 44 08 PM

**Comment by marbemac** [comment]
Good catches! About and mocking page fixed.

**Comment by marbemac** [comment]
👍 will handle those pages in other PRs

**Pull request merged**